applications/MeshMovingApplication/custom_utilities/fixed_mesh_ale_utilities.cpp

```diff
@@ -194,24 +194,20 @@ namespace Kratos
 
         // Search the origin model part nodes in the virtual mesh elements and
         // interpolate the values in the virtual element to the origin model part node
-
-        unsigned int max_results=10000;
-        double search_tol = 1e-5;
-        block_for_each( rOriginModelPart.Nodes(), typename BinBasedFastPointLocator<TDim>::ResultContainerType(max_results),
-            [&]( Node<3>& rNode,
-                 typename BinBasedFastPointLocator<TDim>::ResultContainerType& search_results
-             )
-            {
+        block_for_each(
+            rOriginModelPart.Nodes(),
+            typename BinBasedFastPointLocator<TDim>::ResultContainerType(mSearchMaxResults),
+            [&](Node<3>& rNode, typename BinBasedFastPointLocator<TDim>::ResultContainerType& rSearchResults){
```

Comment:
```suggestion
            [&](auto& rNode, auto& rSearchResults){
```

try this magic :star_struck:

Reply:
Cool! Thanks for the hint.

Reply:
C++14 btw